docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Standard models"
+sidebar_position: 2
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+### Overview
+Standard data models are authored and maintained by Snowplow. Follow below steps to create one.
+
+### 1. Create a warehouse connection
```

Comment:
Not a fan of numbers in headers... Could you remove them? And also make all of these second level headers so that they show up in the right pane?